common/logger/src/libra_logger.rs

```diff
@@ -68,6 +71,14 @@ impl LogEntry {
         let thread_name = thread_name.map(ToOwned::to_owned);
         let message = event.message().map(fmt::format);
 
+        static HOSTNAME: Lazy<Option<String>> = Lazy::new(|| {
+            hostname::get()
+                .ok()
+                .and_then(|name| name.into_string().ok())
+        });
+
+        let hostname = HOSTNAME.clone();
```

Comment:
Nit: Static variables with all caps names, especially since it is Lazy initialized should probably go outside of the function.

Reply:
Generally I would agree with you but this is something that is scoped to this specific function and not intended for use outside similar to the visitor struct definition in the same function.